apps/src/storage/firebaseStorage.js

```diff
@@ -455,6 +456,33 @@ FirebaseStorage.resetForTesting = function() {
   resetConfigForTesting();
 };
 
+FirebaseStorage.copyStaticTable = function(tableName, onSuccess, onError) {
+  return incrementRateLimitCounters()
+    .then(loadConfig)
+    .then(config => {
+      return enforceTableCount(config, tableName);
+    })
+    .then(() => {
+      return getSharedDatabase()
+        .child(`counters/tables/${tableName}`)
+        .once('value');
+    })
+    .then(snapshot => {
+      getProjectDatabase()
+        .child(`counters/tables/${tableName}`)
+        .set(snapshot.val());
+    })
+    .then(() => {
+      return getSharedDatabase()
+        .child(`storage/tables/${tableName}/records`)
+        .once('value');
+    })
+    .then(snapshot => {
+      getRecordsRef(tableName).set(snapshot.val());
+    })
+    .then(onSuccess, onError);
```

Comment:
Should errors be handled per operation?

Reply:
I think just checking for error at the end is fine. At most we might want to show a "something went wrong" dialog to the user if the import fails for whatever reason, but I don't think the granularity of knowing which step failed would necessarily be useful.

We can always add more error cases later, but trying to keep things simple/mvp for now